VRDR/DeathRecord.cs

```diff
@@ -7388,6 +7758,1877 @@ public string EmergingIssue20
+  [PropertyParam("system", "The relevant code system.")]
+  [PropertyParam("display", "The human readable version of this code.")]
+  [FHIRPath("Bundle.entry.resource.where($this is Observation).where(code.coding.code='codedraceandethnicity')", "")]
+  public Dictionary<string, string> HispanicCodeForLiteral
+  {
+      get
+      {
+          if (CodedRaceandEthnicityObs != null)
+          {
+              Observation.ComponentComponent racecode = CodedRaceandEthnicityObs.Component.FirstOrDefault(c => c.Code.Coding[0].Code == "HispanicCodeForLiteral");
+              if (racecode != null && racecode.Value != null && racecode.Value as CodeableConcept != null)
+              {
+                  return CodeableConceptToDict((CodeableConcept)racecode.Value);
+              }
+          }
+          return EmptyCodeableDict();
+      }
+      set
+      {
+          if (CodedRaceandEthnicityObs == null)
+          {
+              CreateCodedRaceandEthnicityObs();
+          }
+          CodedRaceandEthnicityObs.Component.RemoveAll(c => c.Code.Coding[0].Code == "HispanicCodeForLiteral");
+          Observation.ComponentComponent component = new Observation.ComponentComponent();
+          component.Code = new CodeableConcept(CodeSystems.ComponentCode, "HispanicCodeForLiteral", "Hispanic Code For Literal", null);
+          component.Value = DictToCodeableConcept(value);
+          CodedRaceandEthnicityObs.Component.Add(component);
+      }
+  }
+
+  /// <summary>Hispanic Code For Literal  Helper</summary>
+  /// <value>Hispanic Code For Literal Helper.</value>
+  /// <example>
+  /// <para>// Setter:</para>
+  /// <para>ExampleDeathRecord.HispanicCodeForLiteralHelper = VRDR.ValueSets.RaceCode.African ;</para>
+  /// <para>// Getter:</para>
+  /// <para>Console.WriteLine($"Hispanic Code For Literal: {ExampleDeathRecord.HispanicCodeForLiteral}");</para>
+  /// </example>
+  [Property("Hispanic Code For Literal Helper", Property.Types.String, "Coded Content", "Hispanic Code For Literal.", true, IGURL.CodedRaceAndEthnicity, false, 34)]
+  [PropertyParam("code", "The code used to describe this concept.")]
+  [FHIRPath("Bundle.entry.resource.where($this is Observation).where(code.coding.code='codedraceandethnicity')", "")]
+  public string HispanicCodeForLiteralHelper
+  {
+      get
+      {
+          if (HispanicCodeForLiteral.ContainsKey("code"))
+          {
+              return HispanicCodeForLiteral["code"];
+          }
+          return null;
+      }
+      set
+      {
+          SetCodeValue("HispanicCodeForLiteral", value, VRDR.ValueSets.HispanicOrigin.Codes);
+      }
+  }
+
+
+
+
+
+        /// <summary>Entity Axis Cause Of Death</summary>
+        /// <value>Entity-axis codes</value>
+        /// <example>
+        /// <para>// Setter:</para>
+        /// <para>Tuple<string, string, string, string>[] eac = new Tuple<string, string, string, string>{Tuple.Create("value", "position", "line", "code")}</para>
```

Comment:
Might need to somehow quote <string because .NET is getting confused with XML...

Reply:
I'm not sure how to fix this yet ...